Firestore/core/test/unit/bundle/bundle_reader_test.cc

```diff
@@ -325,15 +332,27 @@ TEST_F(BundleReaderTest, ReadsQueryAndDocument) {
       VerifyFullBundleParsed(reader, "bundle-1", testutil::Version(6000004000));
 
   EXPECT_EQ(elements.size(), 4);
-  VerifyNamedQueryEncodesToOriginal(
-      *reinterpret_cast<NamedQuery*>(elements[0].get()), LimitQuery());
-  VerifyNamedQueryEncodesToOriginal(
-      *reinterpret_cast<NamedQuery*>(elements[1].get()), LimitToLastQuery());
-  VerifyDocumentMetadataEquals(
-      *reinterpret_cast<BundledDocumentMetadata*>(elements[2].get()),
-      DocumentMetadata1());
-  VerifyDocumentEncodesToOriginal(
-      *reinterpret_cast<BundleDocument*>(elements[3].get()), Document1());
+  {
+    SCOPED_TRACE("LimitQuery");
```

Comment:
Sorry, looking at this more closely, it's only necessary when invoking the same helper function more than once within the same test. So for example it's necessary in `ReadsQueryAndDocumentWithUnexpectedOrder` below where `VerifyDocumentMetadataEquals` and `VerifyDocumentEncodesToOriginal` are called twice. Here every helper is only called once, so I think it's actually unnecessary. Really sorry for asking for an unnecessary change.

Reply:
We still have two named query check, so it is still useful there right?

Removed other places.

No need to apologize, I now understand better how this is used.